openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -6,6 +6,36 @@ info:
 servers:
   - url: "http://localhost:8080/workflowhub"
 paths:
+  /PaymentIngestionWF/{ingestionFileId}:
+    post:
+      tags:
+        - WorkflowHub
+      operationId: createPaymentIngestionWF
```

let's use the verb used for the WF main method, followed by a workflow context information
```suggestion
      operationId: ingestPaymentsReportingFile
```

---

done